R/write.R

```diff
@@ -20,8 +20,8 @@
 #'   will append to existing file. In both cases, if file does not exist a new
 #'   file is created.
 #' @param col_names Write columns names at the top of the file?
-#' @param delim Delimiter used to separate values. Defaults to `" "`. Must be
-#'   a single character.
+#' @param delim Delimiter used to separate values. Defaults to `" "`. `write_excel_csv()` has `","` as default and
```

Comment:
somewhere in the documentation we should explain why the new funciton is needed, when to use it.

Reply:
okey

Reply:
done

Reply:
I now don't understand this.... do you understand what `Defaults to " ".` refers to?

Reply:
write_delim function has " " has a default parameter.

Reply:
so shouldn't we write sth like this? seems to make more sense

> Defaults to `" "` for  `write_delim`, to `","` for `write_excel_csv` and to `";"` for `write_excel_csv2`